tests/test_translation.py

```diff
@@ -76,6 +82,10 @@ def test_translator(self):
             v1 = ObservationInfo(header, translator_class=InstrumentTestTranslator)
         self.assertEqual(v1.instrument, "SCUBA_test")
         self.assertEqual(v1.telescope, "LSST")
+        self.assertEqual(v1.exposure_id, 22)
+        self.assertEqual(v1.detector_name, "76")
+        self.assertEqual(v1.relative_humidity, 55.0)
+        self.assertIsInstance(v1.relative_humidity, float)
```

Comment:
Should you check explicit types for the other two you added?

Reply:
For the other two, `assertEqual` checks that `22 != "22"` and `"76" != 76`. For the third one `assertEqual` is "clever enough" to let you get away with `55.0 == 55` so you don't get to fully test the right thing has happened.  It's possible, albeit unlikely, that `exposure_id` could be `22.0` so I'll add a type check for that.

Reply:
Sounds good.